lib/ReactViews/Map/MenuBar.jsx

```diff
@@ -26,36 +25,38 @@ const StyledMenuBar = styled.div`
 `;
 // The map navigation region
 const MenuBar = observer((props) => {
+  const viewState = useViewState();
+  const terria = viewState.terria;
   const menuItems = props.menuItems || [];
   const handleClick = () => {
     runInAction(() => {
-      props.viewState.topElement = "MenuBar";
+      viewState.topElement = "MenuBar";
     });
   };
 
-  const storyEnabled = props.terria.configParameters.storyEnabled;
-  const enableTools = props.terria.getUserProperty("tools") === "1";
+  const storyEnabled = terria.configParameters.storyEnabled;
+  const enableTools = terria.userProperties.get("tools") === "1";
```

Comment:
Do you think this is a bit weird, looking for `tools=1`? Should we make it `tools=true`, or just `enableTools` ?

Reply:
We use this pattern a bit (e.g. hideWorkbench=1, which I've just added a quick fix for)